spyder/utils/site/sitecustomize.py

```diff
@@ -555,7 +555,13 @@ def __init__(self, namelist=None, pathlist=None):
             namelist = []
         spy_modules = ['sitecustomize', 'spyder', 'spyderplugins']
         mpl_modules = ['matplotlib', 'tkinter', 'Tkinter']
-        self.namelist = namelist + spy_modules + mpl_modules
+        # Needed to add modules to the UMR blacklist.

```

Comment:
Please rephrase this like

    # Add other, necessary modules to the blacklist

Reply:
Sorry, this is better:

    # Add other, necessary modules to the UMR blacklist